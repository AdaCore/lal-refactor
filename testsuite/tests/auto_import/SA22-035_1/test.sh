auto_import -P default.gpr --source ./src/main.adb --line 4 --column 4
