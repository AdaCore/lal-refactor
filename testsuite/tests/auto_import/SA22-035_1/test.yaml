description: >
  This test checks that the tool is capable of finding declarations in nested packages as well as
  renames for them.

driver: shell_script
