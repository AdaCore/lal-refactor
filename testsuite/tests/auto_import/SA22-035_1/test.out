Available Imports:
Import: A; Qualifier: A
Import: A; Qualifier: A.B
Import: A; Qualifier: A.B.C
Import: A; Qualifier: A.B.C.D
Import: E; Qualifier: E
Import: E; Qualifier: E.B
Import: E; Qualifier: E.B.C
Import: E; Qualifier: E.B.C.D
Import: F; Qualifier: F
Import: F; Qualifier: F.B
Import: F; Qualifier: F.B.C
Import: F; Qualifier: F.B.C.D

Edits:
main.adb
1:1-1:1 with A;

4:4-4:4 A.

